src/main/java/org/opensearch/knn/jni/FaissService.java

```diff
@@ -173,33 +173,26 @@ public static native KNNQueryResult[] queryIndexWithFilter(
     public static native byte[] trainIndex(Map<String, Object> indexParameters, int dimension, long trainVectorsPointer);
 
     /**
+     * <p>
+     * The function is deprecated. Use {@link JNICommons#storeVectorData(long, float[][], long, long)}
```

Comment:
I don't want to remove as of now because I not testing the training feature in this PR at-least. I want to start with deprecation message in this PR and slowly work my way to remove this code.